### PR TITLE
fix for double challenge factor

### PR DESCRIPTION
**Pull request by kabakakao**
These changes had to be made, if you need the stepup-factor. Without these changes the challenge "correctAnswer" isn't shown the second time

## Description
Instead of setting the function `_print_correct_answer` to None this change checks if the answer is the same as in the first attempt. 

## Motivation and Context
In our organization we are using the "correctAnswer" Factor. And the stepup-auth is needed. So the factor came up twice. Without these changes it isn't shown the second time

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by kabakakao** [comment]
@epierce  any chance to get it merged? It is a really annoying issue for us. Please let me know, if I have to change anything.

**Review by epierce — CHANGES_REQUESTED** [review]

File: gimme_aws_creds/okta.py [hunk 1]
```diff
@@ -116,13 +116,15 @@ def use_oauth_id_token(self, val=True):
 
     def stepup_auth(self, embed_link, state_token=None):
         """ Login to Okta using the Step-up authentication flow"""
+        self.ui.info("Additional login is needed")
```
[the remark this answers is not in the document]
Comment (kabakakao) [reply gimme_aws_creds/okta.py:119]: This is just an information, when the stepup_auth is needed.

[the remark this answers is not in the document]
Comment (epierce) [reply gimme_aws_creds/okta.py:119]: Right - but this adds a redundant notification to the user:
```
> gimme-aws-creds
Using password from keyring for eric.pierce@nike.com
Additional login is needed
Multi-factor Authentication required.
```

Also, this change needs to be updated for the current state of the master branch.  The Okta library has been refactored to handle Okta Identity Engine domains.

[the remark this answers is not in the document]
Comment (epierce) [reply gimme_aws_creds/okta.py:119]: Having the 'additional login is needed' notification pop up for everyone doesn't make sense.  If you want that notification when the second MFA challenge happens, you need to build in the logic to show it just in that case. 

Have you looked at the latest version of the project in master?  There have been major changes to the code to handle OIE domains. The functions you've updated have moved to a different file.

[the remark this answers is not in the document]
Comment (kabakakao) [reply gimme_aws_creds/okta.py:119]: Okay. I will take a look at the new master tomorrow. Will you release this new version shortly?

[the remark this answers is not in the document]
Comment (epierce) [reply gimme_aws_creds/okta.py:119]: I'm planning to release v2.6.0 on Monday

**Comment by kabakakao** [comment]
This is Not needed anymore due to Code changes

**Pull request closed without merging**